lms/services/canvas_api/client.py

```diff
@@ -217,6 +224,35 @@ def list_files(self, course_id):
             schema=self._ListFilesSchema,
         )
 
+    def get_course_groups(self, course_id, only_own_groups=True):
```

Comment:
There's more method here than are actually needed but will trim them when everything it's more clear